src/features/A3_admin_upload_age_weight_key.feature

```diff
@@ -26,3 +26,13 @@ Feature: Administrative users can manage paper based returns
     Then  I expect the error summary to show the following errors
       | There is a problem         |
       | The selected file is empty |
+  
+  Scenario: Upload infected file
+    When  I select Dee as the gate for the age weight key
+    And   I enter the year as the current year for the age weight key
+    And   I upload the file eicar.com.csv to the age weight key
+    And   I click upload
+    # If the antivirus is not running a generic error message is shown
```

Comment:
Is this comment here because we're expecting to add another test to cover it later?